Source/UI/IdentityUI.swift

```diff
@@ -136,6 +140,8 @@ public class IdentityUI {
     }
     private let _identityManager: IdentityManager
 
+    private var sharedWebCredentialsController: SharedWebCredentialsController?
```

Comment:
Can we avoid storing it? It should exist during fetching time only

Reply:
We need to retain it as the delegate is weak.

Reply:
But I've changed it to be removed (nilled out) once the completion blocks are done.